proxy/envoy/testdata/ingress-route-2.yaml.golden

```diff
@@ -0,0 +1,13 @@
+destination: foo.default.svc.cluster.local
+match:
+  http_headers:
```

Comment:
what's the difference between the two? can you rename from x-1 and x-2 to something more meaningful?